certified-connectors/MongoDB/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1421 @@
+                },
+                "modifiedCount": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "modifiedcount"
+                }
+              }
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              },
+              "Accept": {
+                "description": "Accept",
+                "type": "string"
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request",
+            "schema": {},
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              },
+              "Accept": {
+                "description": "Accept",
+                "type": "string"
+              }
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {},
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              },
+              "Accept": {
+                "description": "Accept",
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
+    },
+    "/action/deleteOne": {
+      "post": {
+        "summary": "Delete Document",
+        "description": "To delete a single document, use the deleteone endpoint. Use the filter property in the request body to specify the search criteria.If more than one document matches the criteria, only the first match will be deleted.",
+        "operationId": "DeleteDocument",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "x-ms-summary": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/{{CONTENT_TYPE}}",
+            "description": "Content-Type"
+          },
+          {
+            "name": "Access-Control-Request-Headers",
+            "x-ms-summary": "Access-Control-Request-Headers",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "*",
+            "description": "Access-Control-Request-Headers"
+          },
+          {
+            "name": "api-key",
+            "x-ms-summary": "api-key",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "{{API_KEY}}",
+            "description": "api-key"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "dataSource": {
+                  "type": "string",
+                  "description": "datasource",
+                  "title": ""
```

Comment:
@DianaMDB please look through empty titles and add value to them. Value should be a valid sentence, e.g. `Data source`.